sandbox/registration-processor-dmz.properties

```diff
@@ -1,5 +1,8 @@
 ####################################################################################################
 #---------------------------------------- COMMON PROPERTIES ----------------------------------------
+
+registration.processor.main-process=NEW,UPDATE,LOST,RES_UPDATE,ACTIVATE,DEACTIVATE
```

Comment:
can we rename this property to registration.processor.main-processes and move these new properties to just above

#--------------------------------Registration core library properties------------------------------#

Also give proper comments to each new property how it affects the behaviour

Reply:
updated

Reply:
resolved